packages/component-library/components/Text/Text.tsx

```diff
@@ -33,12 +33,13 @@ type TextProps = {
     | "zen-data-small-units"
   inheritBaseline?: boolean
   inline?: boolean
+  children: React.ReactNode
```

Comment:
Probs want to make this optional. Even though I'd probably make it required if I building it now, it's a deprecated component and there's heaps of usages. Better to be on the safe side here and make sure we don't cause a breaking change here.